exotations/task_maps/task_map/include/task_map/Point2Line.hpp

```diff
@@ -48,6 +48,8 @@ class Point2Line : public TaskMap, public Instantiable<Point2LineInitializer>
 
     virtual void update(Eigen::VectorXdRefConst x, Eigen::VectorXdRef phi, Eigen::MatrixXdRef J);
 
+  void setEndPoint(Eigen::Vector3d point);
```

Comment:
C++ formatting not applied either - see documentation on how to do it.

Reply:
FYI, I applied formatting as specified [here](http://ipab-slmc.github.io/exotica/Code-Formatting.html) but this wasn't picked up, reason is because `.hpp` files are not included. 
This 
```
find -name '*.cpp' -o -name '*.h' | xargs clang-format-3.8 -style=file -i
```
should be changed to 
```
find -name '*.cpp' -name '*.h' -o -name '*.hpp' | xargs clang-format-3.8 -style=file -i
```

Reply:
Note, I've corrected docs in latest commit dd80c3a

Reply:
I dont think this should be changed. We do not want headers named .hpp and the Point2Line slipped through. We should rename that header to be in line with all others.